benchmark/Program.cs

```diff
@@ -0,0 +1,72 @@
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Running;
+using LibstapsdtPinvokes;
+
+namespace Benchmark;
+
+[JsonExporterAttribute.Full]
+[JsonExporterAttribute.FullCompressed]
+[SimpleJob(RuntimeMoniker.NativeAot80)]
+public class NotObservedProbeFireBenchmarks
```

Comment:
nit: not + observed = unobserved

```suggestion
public class UnobservedProbeFireBenchmarks
```